_layouts/docu.html

```diff
@@ -82,6 +82,10 @@
 				{% if page.url contains '/docs/' %}
 				<div class="improvethispage"><a href="{% github_edit_link %}" title="Go to GitHub">Page Source</a></div>
 				{% endif %}
+
+				{% if page.url contains '/docs/' %}
```

Comment:
Combine this if block with the above one?

Reply:
good point, thx!